addons/web/controllers/dataset.py

```diff
@@ -13,17 +13,27 @@
 _logger = logging.getLogger(__name__)
 
 
+def _call_kw_readonly(registry, request):
+    params = request.get_json_data()['params']
+    model_class = registry[params['model']]
+    method_name = params['method']
+    for cls in model_class.mro():  # TODO this is maybe not the most efficient way to do that, we could maybe set it on the method during setup
+        method = getattr(cls, method_name, None)
+        if method is not None and hasattr(method, '_readonly'):
+            return method._readonly # should always be True but allows to set a method read write in an override if neccessary
```

Comment:
The comment should appear where `_readonly` is defined, not here.

Reply:
It was more to indicate why we check the value instead of 
`return method is not None and hasattr(method, '_readonly')`
but i'll remove the comment :)